models/comment-models.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../db/connection.js');
+const format = require('pg-format');
+
+exports.deleteComment = (comment_id) => {
+    return db.query(format(`DELETE FROM comments WHERE comment_id = %L RETURNING *;`, comment_id))

```

Comment:
%L is normally used for nested arrays in my experience so $1 syntax would probs be more suitable here but it's up to you, both accomplishing the same thing just worth noting!

Reply:
Thanks Hev, not sure what's going wrong with the $1 syntax but I haven't been able to get it to work?!